test/CMakeLists.txt

```diff
@@ -14,8 +14,6 @@ set(EXTRAFILES
     )
 
 set(PFILES
-    CMakeLists.txt
-    Jamfile
```

Comment:
You can't just remove these two files, they are there for a reason, so they show up in the .vcxproj:
![image](https://user-images.githubusercontent.com/1503976/151041404-a6038bc1-b072-4ca4-8c70-dfced750820b.png)